components/coachmark/index.css

```diff
@@ -158,9 +156,10 @@
 	justify-self: start;
 	color: var(--mod-coachmark-step-color, var(--spectrum-coachmark-step-color));
 	font-size: var(--mod-coachmark-step-font-size, var(--spectrum-coachmark-step-font-size));
-	font-weight: var(--mod-coachmark-step-text-font-weight, var(--spectrum-coachmark-step-text-font-weight));
+	font-weight: var(--mod-coachmark-step-text-font-weight, var(--spectrum-coachmark-step-font-weight));
+	font-family: var(--spectrum-coachmark-step-font-family);
```

Comment:
Making a quick search in our CSS files, sometimes we provide a mod for font-family (for instance `--mod-coachmark-step-font-family`), sometimes not. Is there a rhyme or reason for this difference, and if not, can I safely assume that the "correct" way is not to provide a mod?

Reply:
I opt not to create a mod at all unless requested but only because I want to avoid adding more to our APIs than is necessary.